todo_list_app/urls.py

```diff
@@ -5,12 +5,10 @@
 router = DefaultRouter(trailing_slash=False)
 
 router.register(r'categories', views.CategoryViewSet, basename='category')
+router.register(r'tasks', views.TaskViewSet, basename='task')
```

Comment:
What is the purpose of `r` here?

Reply:
By default string takes \ -> backslash as escape character for exmaple if you want to add " -> double quotes in string the interpreter may take it as end of string and gives error on further characters e.g "hello "world"" so we '\' to tackle this "hello \"world\""

'r' at the front of string mark it as raw string in order to take '\' as character not as escape character